app/locale/et_EE/Sendsmaily_Sync.csv

```diff
@@ -26,3 +26,11 @@
 "Module is not activated!", "Smaily moodul ei ole aktiveeritud"
 "Could not validate credentials! Your current input is saved, but the module is still disabled. Please try again!", "Ei õnnestunud kasutajat valideerida! Praegused andmed on salvestatud, kuid moodul on endiselt deaktiivne. Palun proovi uuesti!"
 "General settings","Üldised sätted"
+"Newsletter form", "Uudiskirja liitumise vorm"
+"Collect newsletter subscribers from newsletter form. Learn more from our <a href=http://help.smaily.com/en/support/solutions/articles/16000092458-creating-automation-workflows#a4 target=_blank>tutorial how to create welcome emails.</a>","Kogu uudiskirjaga liitujaid Smailysse otse uudiskirja vormilt. Loe kuidas saad saata<a href=http://help.smaily.com/en/support/solutions/articles/16000092458-creating-automation-workflows#a4 target=_blank> automaatse tervituskirja liitujale.</a>"
```

Comment:
You can escape double quotes in CSV by having another quote next to it. This should be a valid CSV syntax:
```
"Learn more from <a href=""http://..."" target=""_blank"">tutorial</a>","Learn more from <a href=""http://..."" target=""_blank"">tutorial</a>"
```

Reply:
Cool db5f1a6025fc752c7739437d38b3644d68299912